view/theme/frio/templates/settings_features.tpl

```diff
@@ -19,7 +19,11 @@
 				<div id="{{$g}}-settings-content" class="panel-collapse collapse" role="tabpanel" aria-labelledby="{{$g}}-settings-title">
 					<div class="section-content-tools-wrapper">
 						{{foreach $f.1 as $fcat}}
+							{{* Here is an addional condition to filter the richtext feature
+							because frio doesn't support the richtext *}}
+							{{if $fcat.0 != 'feature_richtext'}}
```

Comment:
Yes I have also think about this. But this message wouldn't be translatable. Or we put it in the core. But to put a unsupported message in the core which is used only by one theme wouldn't be a good solution